neqo-transport/src/cc/classic_cc.rs

```diff
@@ -614,6 +614,7 @@ mod tests {
     };
 
     const IP_ADDR: IpAddr = IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0));
+    const MTU: usize = 1_500;
```

Comment:
Given that each `mod tests` in `neqo-transport/src/cc/` needs `MTU`, how about defining it only once at a higher level?